LuckParser/Models/BossLogic/BossLogic.cs

```diff
@@ -522,5 +521,11 @@ protected static List<CombatItem> GetFilteredList(ParsedLog log, long skillID, u
             }
             return filtered;
         }
+
+        public static int getRotationFromFacing(Point3D facing)
```

Comment:
I think this method is better off being in Point3D.cs. It looks out of place in boss logic

Reply:
True, moved it.